README.md

```diff
@@ -41,9 +53,10 @@ Its features comprise
 * IPv6
 * 6LoWPAN (RFC4944, RFC6282, and RFC6775)
 * UDP
-* RPL (storing mode)
+* RPL 
 * CoAP
 * CCN-Lite
+* ...
```

Comment:
Not sure if dot dot dot looks nice, if the list begins with `...features including, but not limited to:` (: